addons/account/models/account_invoice.py

```diff
@@ -1045,6 +1045,7 @@ def refund(self, date_invoice=None, date=None, description=None, journal_id=None
                 'in_invoice': ('vendor bill refund')}
             message = _("This %s has been created from: <a href=# data-oe-model=account.invoice data-oe-id=%d>%s</a>") % (invoice_type[invoice.type], invoice.id, invoice.number)
             refund_invoice.message_post(body=message)
+            invoice.refund_invoice_id = refund_invoice
```

Comment:
we add this field because currently we does't have any connection for invoice refund.

so for taking refund from invoice need this field.